plugins/nodes/src/js/components/__tests__/NodesTable-test.js

```diff
@@ -1,6 +0,0 @@
-/* eslint-disable no-unused-vars */
```

Comment:
These changes were pulled from master. I didn't manually remove this file

Reply:
if so, please remove them from your commit 🤔